tests/cli/test_cli_regression_mol.py

```diff
@@ -223,3 +223,29 @@ def test_freeze_model(monkeypatch, data_path, model_path, tmp_path):
     assert torch.equal(
         trained_model.predictor.ffn[0][0].weight, frzn_model.predictor.ffn[0][0].weight
     )
+
+
```

Comment:
```suggestion
import platform
@pytest.mark.skipif(platform.python_version_tuple()[1] != '11', reason="ray does not support python 3.12+")
```

Reply:
could also just `from chemprop.cli.hyperopt import NO_HYPEROPT, NO_OPTUNA` and change this to `@pytest.mark.skipif(NO_HYPEROPT or NO_OPTUNA, reason="hyperopt dependencies not installed")`

Reply:
The second one is better

Reply:
Thanks for the suggestion! I have added this.